graphql-schemas/schema.yml

```
@@ -1092,19 +1092,23 @@
   fields:
   - { name: name, type: string, isRequired: true }
   - { name: default, type: boolean, isRequired: true }
-  - { name: weights, type: NamespaceTerraformResourceALBTargetWeights_v1, isRequired: true }
   - { name: ips, type: string, isList: true }
   - { name: openshift_service, type: string }
 
-- name: NamespaceTerraformResourceALBTargetWeights_v1
+- name: NamespaceTerraformResourceALBRules_v1
   fields:
-  - { name: read, type: int, isRequired: true }
-  - { name: write, type: int, isRequired: true }
+  - { name: condition: type: NamespaceTerraformResourceALBConditon_v1, isRequired: true}
+  - { name: action: type: NamespaceTerraformResourceALBAction_v1, isRequired: true, isList: true}
 
-- name: NamespaceTerraformResourceALBPaths_v1
+- name: NamespaceTerraformResourceALBConditon_v1
   fields:
-  - { name: read, type: string, isList: true }
-  - { name: write, type: string, isList: true }
+  - { name: path: type: string, isRequired: true}
+  - { name: methods: type: string, isRequired: true}
+
+- name: NamespaceTerraformResourceALBAction_v1
+  fields:
+  - { name: target: type: string, isRequired: true}
+  - { name: weight: type: int, isRequired: true}
```

```suggestion
  - { name: path, type: string, isRequired: true}
  - { name: methods, type: string, isRequired: true}

- name: NamespaceTerraformResourceALBAction_v1
  fields:
  - { name: target, type: string, isRequired: true}
  - { name: weight, type: int, isRequired: true}
```

more `:`s instead of `,`s :)